charts/amoro/templates/_optimizer.tpl

```diff
@@ -46,6 +60,21 @@ properties:
   {{- end -}}
 {{- end -}}
 
+{{- define "amoro.optimizer.container.spark" -}}
+container-impl: com.netease.arctic.server.manager.SparkOptimizerContainer
+properties:
+  job-uri: {{ .Values.optimizer.spark.image.jobUri | quote }}
+  ams-optimizing-uri: {{include "amoro.svc.optimizing.uri" . | quote}}
+  spark-home: /opt/spark
+  export.SPARK_HOME: /opt/spark
+  spark-conf.spark.kubernetes.container.image: {{ include "amoro.optimizer.container.spark.image" .  | quote }}
+  spark-conf.spark.kubernetes.authenticate.driver.serviceAccountName: {{ include "amoro.sa.name" . | quote }}
+  spark-conf.spark.kubernetes.namespace: {{ include "amoro.namespace" . | quote }}
```

Comment:
Does this parameter cause the optimizer and AMS to be in the same namespace?
In fact, AMS and Optimizer should usually belong to different namespaces.

Reply:
OK, I will divide them into two different namespaces.